projects/media-viewer/src/lib/annotations/rectangle/rectangle.component.ts

```diff
@@ -15,8 +16,25 @@ export class RectangleComponent {
   @Input() draggable = true;
 
   @Output() click = new EventEmitter();
+  @Output() update = new EventEmitter<Rectangle>();
 
   onClick() {
     this.click.emit();
   }
+
+  onMove(el: HTMLElement) {
+    const [left, top] = el.style.transform.match(/\d+/g);
+
+    this.rectangle.x = +left / this.zoom;
+    this.rectangle.y = +top / this.zoom;
```

Comment:
should these be multiplications instead of divisions?

Reply:
no, we're scaling it back up to full size (dividing by a fraction makes things bigger)